### PR TITLE
Various minor fixes made in the course of writing a SLURM job submission script

### PR DESCRIPTION
**Pull request by gabbard**

### CODE REVIEW

File: vistautils/logging_utils.py [hunk 1]
```diff
@@ -60,8 +61,13 @@ def _config_logging_from_params(params):
     logging.getLogger().setLevel(level)
 
     # configure a console handler with a default formatter. We could make this
-    # configurable in the future
-    console_handler = logging.StreamHandler()
+    # configurable in the future.
+    console_handler = logging.StreamHandler(
+        # By default, Python will send log messages to stderr,
+        # but since most of our code is run on a cluster where stdout and stderr
+        # are routed to separate files, keeping things on stdout is easier.
```
Comment (berquist) [inline vistautils/logging_utils.py]: If you don't specify `--error`, SLURM will send stderr to stdout, no need to specify `-j y` like under UGE.

File: vistautils/memory_amount.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+from attr import attrib, attrs
+from attr.validators import and_, in_, instance_of
+
+from immutablecollections import immutabledict
+
+from vistautils.range import Range
+
+
+class MemoryUnit(Enum):
```
Comment (berquist) [inline vistautils/memory_amount.py]: I think we always want the `unique` decorator: https://docs.python.org/3/library/enum.html#enum.unique

File: vistautils/logging_utils.py (hunk 1 above)
Comment (gabbard) [reply vistautils/logging_utils.py]: Aha! Thanks.  I think we still want the change in behavior but I will shift the justification.

File: vistautils/memory_amount.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    """
+    A unit in which memory can be measured.
+    """
+
+    KILOBYTES = auto()
```
Comment (berquist) [inline vistautils/memory_amount.py]: If you're ok with directly tying the abstract and string representations together, you can replace `auto()` with `"K"`, `"M"`, and so on, then you'd be able to get rid of `_STRING_TO_UNIT`.

**Pull request merged**